t/40-script_load_dump_templates.t

```diff
@@ -33,6 +33,13 @@ sub dump_templates {
 
 sub decode { Cpanel::JSON::XS->new->relaxed->decode(path(shift)->slurp); }
 
+sub check_property {
+    my ($schema, $table, $property, $values) = @_;
```

Comment:
It would be nice to use signatures in new code:
```suggestion
sub check_property ($schema, $table, $property, $values) {
```

Reply:
I kinda figured it was better to be consistent - within a single script either *everything* should use signatures or *nothing* should. but if you'd prefer mixed I can do that.

Reply:
Mixing is of course leading to inconsistency. However, we want to move forward with signatures and the easiest way to do this is (with the least additional effort and the least conflict potential) is to simply use signatures when adding/changing affected lines anyway.